internal/integration/knot/network/amqp.go

```diff
@@ -161,12 +163,38 @@ func (a *AMQP) PublishPersistentMessage(exchange, exchangeType, key string, data
 
 func (a *AMQP) notifyWhenClosed() {
 	errReason := <-a.conn.NotifyClose(make(chan *amqp.Error))
+
+	reconnectionBackOff := backoff.NewExponentialBackOff()
+	reconnectionBackOff.InitialInterval = 30 * time.Second
+	reconnectionBackOff.MaxInterval = 5 * time.Minute
+	reconnectionBackOff.Multiplier = 1.7
```

Comment:
Why do you need to multiply by 1.7? Specify this constant and the others on line 168 and 169.